go/provisioner/local/local.go

```diff
@@ -798,6 +801,8 @@ func (p *localProvisioner) provisionMultipooler(ctx context.Context, req *provis
 		"--pg-port", fmt.Sprintf("%d", pgPort),
 		"--hostname", "localhost",
 		"--pgbackrest-stanza", "multigres",
+		"--connpool-admin-password", "postgres", // Password created in initializePgctldDirectories
```

Comment:
cc @rafael re: thoughts on passwords, since https://github.com/multigres/multigres/pull/342 relates to that

I don't know if a password for the unix socket is useful, but if it is then setting it via command line argument in production doesn't feel particularly secure since it'd leak via things like `ps` output.